v2/federation/watch_test.go

```diff
@@ -41,54 +38,44 @@ func TestWatchBundle_OnUpate(t *testing.T) {
 		latestBundle: &spiffebundle.Bundle{},
 	}
 
-	err := WatchBundle(ctx, td, "some url", watcher)
+	err := federation.WatchBundle(ctx, td, be.FetchBundleURL(), watcher, federation.WithWebPKIRoots(be.RootCAs()))
 	assert.Equal(t, 2, watcher.onUpdateCalls)
 	assert.Equal(t, 0, watcher.onErrorCalls)
 	assert.Equal(t, context.Canceled, err)
 }
 
 func TestWatchBundle_OnError(t *testing.T) {
-	fetchErr := errs.New("oops...I did it again...")
-	fetchBundleCallback = func(ctx context.Context, trustDomain spiffeid.TrustDomain, url string, option ...FetchOption) (*spiffebundle.Bundle, error) {
-		return nil, fetchErr
-	}
-
-	td := spiffeid.RequireTrustDomainFromString("domain.test")
 	ctx, cancel := context.WithCancel(context.Background())
 	watcher := &fakewatcher{
 		t:            t,
 		nextRefresh:  1 * time.Second,
-		expectedErr:  fetchErr.Error(),
+		expectedErr:  `federation: could not GET bundle: Get "wrong%20url": unsupported protocol scheme ""`,
```

Comment:
This is broken in CI/CD because of the difference in the way this error is formatted in go1.13 v.s. go1.14...